backend/ibutsu_server/widgets/jenkins_heatmap.py

```diff
@@ -35,8 +35,10 @@ def _calculate_slope(x_data):
     return slope
 
 
-def _get_builds(job_name, builds, project=None):
+def _get_builds(job_name, builds, project=None, custom_filter=None):
```

Comment:
Oh, I see why you used `custom_filter`. Technically we could translate it (`filter` => `filter_`) just before we `**kwargs` it. What do you think?

Reply:
Yeah I decided to call it `custom_filter` to differentiate it from the other parameters, which are really just specific filters. Maybe `additional_filters` would be more appropriate?

Reply:
Sounds good to me.